docker-dist/Dockerfile-wf-agent-domain

```diff
@@ -31,15 +32,15 @@ EXPOSE 8080 9090
 
 WORKDIR /tmp
 
-RUN java -jar installer.jar --enabled true --server-url ${HAWKULAR_SERVER} \
-  --module-dist classpath:hawkular-wildfly-agent-wf-extension.zip --username ${HAWKULAR_USER} \
+RUN java -jar installer.jar --enabled true --server-url '${env.HAWKULAR_SERVER_PROTOCOL}://${env.HAWKULAR_SERVER_ADDR}:${env.HAWKULAR_SERVER_PORT}' \
+  --module-dist classpath:hawkular-wildfly-agent-wf-extension.zip --username '${env.HAWKULAR_AGENT_USER}' \
   --target-config=/opt/jboss/wildfly/domain/configuration/host.xml \
-  --password ${HAWKULAR_PASSWORD} --target-location /opt/jboss/wildfly
+  --password '${env.HAWKULAR_AGENT_PASSWORD}' --target-location /opt/jboss/wildfly
 
 USER root
 RUN chown -R jboss:0 /opt/jboss/wildfly/domain && \
     chmod -R ug+rw /opt/jboss/wildfly/domain && \
     rm /tmp/installer.jar
 
-USER JBOSS
+USER jboss
 CMD /opt/jboss/wildfly/bin/domain.sh
```

Comment:
We need immutable/in-container flag in a domain.conf unless I am missing something (again) :)